src/MaterialDesignControls/ControlsMaterial3/MaterialEntry.cs

```diff
@@ -35,6 +35,8 @@ public MaterialEntry()
                     this.txtEntry.Focus();
                 }
             };
+
+            this.Label.GestureRecognizers.Clear();
             this.FrameContainer.GestureRecognizers.Add(frameTapGestureRecognizer);
```

Comment:
Should be "this.Label.GestureRecognizers.Add"? As I can see the rest of controls add it to the label instead to the frame